spec/std/time/location_spec.cr

```diff
@@ -0,0 +1,323 @@
+require "spec"
+require "./spec_helper"
+
+class Time::Location
+  def __cached_range
+    @cached_range
+  end
+
+  def __cached_zone
+    @cached_zone
+  end
+
+  def __cached_zone=(zone)
+    @cached_zone = zone
+  end
+
```

Comment:
A small nitpick here: can we close `Time::Location` here and avoid nesting the specs into it?

Thank you.

Reply:
Some examples use protected methods like `transitions` to ensure that the spec covers all expected behaviour. I think it's easier to wrap the specs in the class namespace than providing additional accessors. At least I can't find any pressing argument against it.

If it was easily avoidable, I sure wouldn't do it that way.

Reply:
Thank you for taking the time and provide the details behind it. While I would prefer to avoid testing internals, I believe those can be later improved, so 👍 with current implementation.

Cheers.